specs/receive_funds/02_user_interface.rst

```diff
@@ -0,0 +1,56 @@
+==========================================================
+User interface
+==========================================================
+
+=====  =============  =========  ==========
+ 68        title      tschubotz   created
+=====  =============  =========  ==========
+`68`_  Receive funds  tschubotz  2019-01-17
+=====  =============  =========  ==========
+
+.. _68: https://github.com/gnosis/safe/issues/68
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Receive funds
+-------------
+
+Receive screens that opens when e.g. tapping the
+identicon on the funds overview screen.
+
+- Tapping "Share" opens the OS share sheet.
+- Normal tap & long tap on the address copies it
+  to the clipboard.
+
+.. raw:: html
+
+   <img src="screens/android/receive funds.png" width="320px"/>
```

Comment:
I realized that a border around the image would be nice ... not sure how to do this easily :/

Also would prefer not to have spaces in the file names.

Reply:
Will remove the whitespaces.

Will also add a border. It's raw html, so we can do all the things 😎 . If the others agree we can add it to the feature template

Reply:
Actually, it didn't work :-/

- with the css style attr "border", github overwrites that in the output html.
- the html img attr "border" is ignored also.